i18n/en.toml

```diff
@@ -183,6 +186,9 @@ other = "Objectives"
 [options_heading]
 other = "Options"
 
+[parentoptions_heading]
+other = "Options inherited from parent commands"
```

Comment:
not a blocker... but this change is not closely related to the subject of this PR?

Reply:
To be complete, I added the `examples` and `parentoptions`.
I can remove the `parentoptions` heading as the case for using this string is very low.
Thanks for reviewing.